packages/core-suggest/core-suggest.js

```diff
@@ -264,25 +263,25 @@ function onMutation (self, mutations) {
             // Notify screen readers when visible and list has content
             clearTimeout(self._ariaLiveDelay) // Clear existing delay
```

Comment:
Just tripple-checking - you guy's figured out this was not possible solving without traversing mutations? :)

Reply:
It was possible to hook into the `onInput` -function for parts of this, but counting suggestions proved flaky and inconsistent. Also support would be lost for users that don't use the built-in filtering. With regards to your [previous comment](https://github.com/nrkno/core-components/pull/644#discussion_r853750576) on the subject I suspect I may have misunderstood your question?

Reply:
:) not thinking of moving it into the onInput, and maybe I'm completely off, but it looks to me like all the information needed to build the notifications is already available in the onMutation-function, without traversing the mutation-object. Should we have a co-coding hour just to double check, or should I make a branch testing the hypothesis?